run_all_tests.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-sbt clean scalastyle coverage test it:test coverageReport dependencyUpdates
+sbt clean coverage test it:test coverageReport dependencyUpdate
```

Comment:
"sbt-updates"

Reply:
Added sbt-updates back and undid this change. Tested and get updates report back